SokolTextGame/Commands.cs

```diff
@@ -36,10 +36,24 @@ public class Look : ICommand
     {
         public void Execute(string[] words, World world)
         {
-            if (string.Join(" ", words) == "look") Console.Write("Look... What's next? You can look at your weapon\n");
+            var locObj = world.CurrentLocation().ObjectOnLocation;
+            if (string.Join(" ", words) == "look") Console.Write("Look... What's next? You can look at your weapon or look around\n");
+            else if (string.Join(" ", words.Take(2)) == "look at" && locObj?.Name == string.Join(" ", words.Skip(2))) Console.Write(locObj.Description);
+            else if ((words.Length > 2 && string.Join(" ", words.Take(2)) == "look at") && (locObj?.Name != string.Join(" ", words.Skip(2))))
+            {
+                Console.WriteLine("I don't see this object on location");
+            }
             else if (string.Join(" ", words) == "look at") Console.Write("Look at... what?\n");
             else if (string.Join(" ", words) == "look at my weapon") Console.Write(world?.player?.Weapon.Description);
-            else Console.Write("I know the command \"look at my weapon\"\n");
+            else if (string.Join(" ", words) == "look around")
+            {
+                if (string.IsNullOrEmpty(string.Join(" ", locObj)))
+                {
+                    Console.WriteLine("There's nothing to see in this place.");
+                }
+                else Console.WriteLine($"You see: {string.Join(" ", locObj.Name)}");
+            }
+            else Console.Write("I know the command: \"look at my weapon\" \"look around\"\n");
```

Comment:
This is grammatically incorrect and incomplete. Please change this string to:
`I know commands "look at my weapon", "look around" and "look at <object>"`.